src/kv/store.h

```diff
@@ -327,6 +346,17 @@ namespace kv
         last_committable = v;
       }
 
+      if (h)
+      {
+        // For now, this is not atomic as a new transaction is created in
```

Comment:
I think this is fine for the foreseeable future, there's no clear requirement to make snapshot application atomic at this point.

Reply:
I've removed the comment.